Adafruit_SCD30.h

```diff
@@ -0,0 +1,165 @@
+  0x0010 ///< Command to start continuous measurement
+#define SCD30_CMD_STOP_MEASUREMENTS 0x0104 ///< Command to stop measurements
+#define SCD30_CMD_SET_MEASUREMENT_INTERVAL                                     \
+  0x4600                                ///< Command to set measurement interval
+#define SCD30_CMD_GET_DATA_READY 0x0202 ///< Data ready reg
+#define SCD30_CMD_AUTOMATIC_SELF_CALIBRATION                                   \
+  0x5306 ///< enables/disables auto calibration
+#define SCD30_CMD_SET_FORCED_RECALIBRATION_REF                                 \
+  0x5204 ///< Forces calibration with given value
+#define SCD30_CMD_SET_TEMPERATURE_OFFSET 0x5403 ///< Specifies the temp offset
+#define SCD30_CMD_SET_ALTITUDE_COMPENSATION                                    \
+  0x5102                               ///< Specifies altitude offset
+#define SCD30_CMD_SOFT_RESET 0xD304    ///< Soft reset!
+#define SCD30_CMD_READ_REVISION 0xD100 ///< Firmware revision number
+///////////////////////////////////////////////////////////////
+/**
+ * @brief
+ *
+ * Allowed values for `setDataRate`.
+ */
+typedef enum {
+  SCD30_RATE_ONE_SHOT,
+  SCD30_RATE_1_HZ,
+  SCD30_RATE_7_HZ,
+  SCD30_RATE_12_5_HZ,
+  SCD30_RATE_25_HZ,
+} scd30_rate_t;
+
+class Adafruit_SCD30;
+
+/** Adafruit Unified Sensor interface for temperature component of SCD30 */
+class Adafruit_SCD30_Temp : public Adafruit_Sensor {
+public:
+  /** @brief Create an Adafruit_Sensor compatible object for the temp sensor
+      @param parent A pointer to the SCD30 class */
+  Adafruit_SCD30_Temp(Adafruit_SCD30 *parent) { _theSCD30 = parent; }
+  bool getEvent(sensors_event_t *);
+  void getSensor(sensor_t *);
+
+private:
+  int _sensorID = 0xC02;
+  Adafruit_SCD30 *_theSCD30 = NULL;
+};
+
+/** Adafruit Unified Sensor interface for the humidity sensor component of SCD30
+ */
+class Adafruit_SCD30_Humidity : public Adafruit_Sensor {
+public:
+  /** @brief Create an Adafruit_Sensor compatible object for the humidity sensor
+      @param parent A pointer to the SCD30 class */
+  Adafruit_SCD30_Humidity(Adafruit_SCD30 *parent) { _theSCD30 = parent; }
+  bool getEvent(sensors_event_t *);
+  void getSensor(sensor_t *);
+
+private:
+  int _sensorID = 0xC02 + 1;
+  Adafruit_SCD30 *_theSCD30 = NULL;
+};
+
+/*!
+ *    @brief  Class that stores state and functions for interacting with
+ *            the SCD30 eCO2, Temperature, and Humidity sensor
+ */
+class Adafruit_SCD30 {
+public:
+  Adafruit_SCD30();
+  ~Adafruit_SCD30();
+
+  bool begin(uint8_t i2c_addr = SCD30_I2CADDR_DEFAULT, TwoWire *wire = &Wire,
+             int32_t sensor_id = 0);
+
+  void reset(void);
+  bool dataReady(void);
+  bool getEvent(sensors_event_t *humidity, sensors_event_t *temp);
+
+  bool read(void);
+
+  uint16_t getMeasurementInterval(void);
+  bool setMeasurementInterval(uint16_t interval);
+
+  bool selfCalibrationEnabled(void);
+  bool selfCalibrationEnabled(bool);
```

Comment:
same function name

Reply:
This was meant to be an overload, to allow it to work as a getter or setter, but I can change the API to better match the other accessors

Reply:
make one `enableCalibration` and the other `isCalibrationEnabled`